src/main/java/ac/grim/grimac/utils/collisions/datatypes/SimpleCollisionBox.java

```diff
@@ -20,6 +20,7 @@ public class SimpleCollisionBox implements CollisionBox {
     public double minX, minY, minZ, maxX, maxY, maxZ;
     private boolean isFullBlock = false;
 
+    // If your min/max values are > 1 you should probably check out HexCollisionBox
```

Comment:
shouldn't this comment be on the constructor with min/max parameters?

also use javadoc, might need to import `HexCollisionBox` to use `@link`
```suggestion
    /**
     * If your min/max values are > 1 you should probably check out {@link HexCollisionBox}
     */
```